lib/Client/MultiCurl.php

```diff
@@ -90,8 +96,23 @@ public function proceed(): void
             return;
         }
 
-        if (!$this->curlm && false === $this->curlm = curl_multi_init()) {
-            throw new ClientException('Unable to create a new cURL multi handle');
+        if (!$this->curlm) {
+            if (false === $this->curlm = curl_multi_init()) {
+                throw new ClientException('Unable to create a new cURL multi handle');
+            }
+
+            $cb = function ($parent, $pushed, $headers) {
+                curl_setopt($pushed, CURLOPT_RETURNTRANSFER, true);
+                curl_setopt($pushed, CURLOPT_HEADER, true);
+                curl_setopt($pushed, CURLOPT_HEADERFUNCTION, null);
+                curl_setopt($pushed, CURLOPT_WRITEFUNCTION, null);
```

Comment:
These options do not seam to have much effect.

Doing curl_setopt($pushed, CURLOPT_WRITEFUNCTION, null); does not help...